azkaban-common/src/main/java/azkaban/executor/mail/DefaultMailCreator.java

```diff
@@ -66,12 +66,12 @@ public boolean createFirstErrorMessage(ExecutableFlow flow,
     if (emailList != null && !emailList.isEmpty()) {
       message.addAllToAddress(emailList);
       message.setMimeType("text/html");
-      message.setSubject("Flow '" + flow.getFlowId() + "' has encountered a failure on "
-          + azkabanName);
+      message.setSubject("Flow '" + flow.getFlowId()
```

Comment:
i think we can have those strings as constant variable in this class and use string.format